_includes/index-docs.html

```diff
@@ -3,162 +3,149 @@
-        <p>Short and focused exercises to get you going quickly.</p>
-      </div>
-      <div class="grid-wrapper">
-        {% for guide in values %}
-        {% include index-doc-item.html class="tutorialbkg" docversion=docversion
-          title=guide.title url=guide.url summary=guide.summary
-          keywords=guide.keywords categories=guide.categories origin=guide.origin %}
-        {% endfor %}
-      </div>
-    </div>
-    {% endif %}
-    {% include index-docs-merge.html type="howto" %}
-    {% if values %}
-    <div class="grid__item width-12-12 docslist">
-      <div class="doclist-header">
-        <h3 id="guide">How-to Guides</h3>
-        <p>Step-by-step guides to covering key tasks, real world operations and common problems.</p>
-      </div>
-      <div class="grid-wrapper">
-        {% for guide in values %}
-        {% include index-doc-item.html class="howtobkg" docversion=docversion
-          title=guide.title url=guide.url summary=guide.summary
-          keywords=guide.keywords categories=guide.categories origin=guide.origin %}
-        {% endfor %}
-      </div>
+    <div>
+        <h1 class="title">{{ page.title }}</h1>
     </div>
-    {% endif %}
-    {% include index-docs-merge.html type="concepts" %}
-    {% if values %}
-    <div class="grid__item width-12-12 docslist">
-      <div class="doclist-header">
-        <h3 id="concept">Concepts</h3>
-        <p>Explanations of some of the larger concepts and technologies involved with Quarkus.</p>
-      </div>
-      <div class="grid-wrapper">
-        {% for guide in values %}
-        {% include index-doc-item.html class="conceptsbkg" docversion=docversion
-          title=guide.title url=guide.url summary=guide.summary
-          keywords=guide.keywords categories=guide.categories origin=guide.origin %}
-        {% endfor %}
-      </div>
-    </div>
-    {% endif %}
-    {% include index-docs-merge.html type="reference" %}
-    {% if values %}
-    <div class="grid__item width-12-12 docslist">
-      <div class="doclist-header">
-        <h3 id="reference">References</h3>
-        <p>Technical Resource that covers tools, components, and commands. The encyclopedia for Quarkus.</p>
-      </div>
-      <div class="grid-wrapper">
-        <div class="grid__item docs-card pdfbkg">
-          <h4><a href="https://lordofthejars.github.io/quarkus-cheat-sheet/" target="_blank">Quarkus Cheat Sheet</a></h4>
-          <div class="description">Download full cheatsheet as PDF. <a href="https://developers.redhat.com/search?t=quarkus&f=type%7Echeat_sheet" target="_blank">
-            Get more cheatsheets on the Red Hat Developers website. <i class="fas fa-external-link-alt"></i></a></div>
-          <div class="keywords"></div>
-          <div class="categories"></div>
+
+    <!-- Static content displayed when there is no input in the search form or Javascript is disabled,
+         but also used as a source of data for cards displayed by the vue.js app. -->
```

Comment:
Looks like your reformatted this file from 2-space indent to 4-space indent, and it's making the diff rather hard to read :(